packages/drivers/routerlicious-driver/src/documentServiceFactory.ts

```diff
@@ -90,15 +90,39 @@ export class RouterliciousDocumentServiceFactory implements IDocumentServiceFact
             this.driverPolicies.enableRestLess,
             resolvedUrl.endpoints.ordererUrl,
         );
+
         // the backend responds with the actual document ID associated with the new container.
-        const documentId = await ordererRestWrapper.post<string>(
+        const res = await ordererRestWrapper.post<{ id: string, token: string } | string>(
```

Comment:
See other comment, but expecting token to be possibly undefined here would be great. Looks like the logic below can be left the same.